master/networking/mtu.md

```diff
@@ -3,127 +3,138 @@ title: Configuring MTU
 
-Restarting the `{{site.nodecontainer}}` pods will also update any {{site.prodname}}
-tunnel network interfaces on that node. From this point forward, any pods
-started will also have the updated MTU value.
+**Common MTU sizes**
 
-### Setting tunnel MTU with a Felix environment variable
+| Network MTU            | {{site.prodname}} MTU | {{site.prodname}} MTU with IP-in-IP (IPv4) | {{site.prodname}} MTU with VXLAN (IPv4) |
+| ---------------------- | --------------------- | ------------------------------------------ | --------------------------------------- |
+| 1500                   | 1500                  | 1480                                       | 1450                                    |
+| 9000                   | 9000                  | 8980                                       | 8950                                    |
+| 1460 (GCE)             | 1460                  | 1440                                       | 1410                                    |
+| 9001 (AWS Jumbo)       | 9001                  | 8981                                       | 8951                                    |
+| 1450 (OpenStack VXLAN) | 1450                  | 1430                                       | 1400                                    |
 
-Passing in the environment variable `FELIX_IPINIPMTU` (or `FELIX_VXLANMTU`) when running the
-`{{site.nodecontainer}}` container will set the MTU for Felix to use.
+##### Recommended MTU for overlay networking
 
-When using the Kubernetes self-hosted manifests, Felix derives the IP-in-IP value from
-the `veth_mtu` field of the calico-config ConfigMap, which is set to `1440` by default.
+The extra overlay header used in IP in IP and VXLAN protocols, reduces the minimum MTU by the size of the header. (IP in IP uses a 20-byte header, and VXLAN uses a 50-byte header). Therefore, we recommend the following:
 
-### Setting tunnel MTU with calicoctl
+- If you use VXLAN anywhere in your pod network, configure MTU size as “physical network MTU size minus 50”. 
+- If you use only IP in IP, configure MTU size as “physical network MTU size minus 20”
+- Set the workload endpoint MTU and the tunnel MTUs to the same value (so all paths have the same MTU)
 
-To set the IP-in-IP MTU value for all {{site.prodname}} nodes in your cluster, use the
-following command to retrieve the current Felix settings.
+##### MTU for flannel networking
 
-```bash
-calicoctl get felixconfig default --export -o yaml > felix.yaml
-```
+When using flannel for networking, the MTU for network interfaces should match the MTU of the flannel interface. If using flannel with VXLAN, use the “Calico MTU with VXLAN” column in the table above for common sizes. 
+
+#### Configure MTU for workloads
+  
+When you set the MTU it applies to new workloads. To apply MTU changes to existing workloads, you must restart calico nodes. Restarting the calico/node pods also updates any Calico tunnel network interfaces on that node. 
```

Comment:
> To apply MTU changes to existing workloads, you must restart calico nodes

This isn't correct - to apply MTU to existing workloads, you need to restart those workloads.

Reply:
> Restarting the calico/node pods also updates any Calico tunnel network interfaces on that node. 

You don't need to restart calico/node to update the tunnel interface.

Reply:
Fixed.